opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -553,6 +553,10 @@ def set_attribute(self, key: str, value: types.AttributeValue) -> None:
             with self._lock:
                 self.attributes[key] = value
 
+    def get_attribute(self, key: str) -> types.AttributeValue:
```

Comment:
maybe overkill, but should a unit test be added?

Reply:
Removed that method.